templates/vue-demo-store/components/checkout/CheckoutCartItem.vue

```diff
@@ -37,9 +37,15 @@ const updateQuantity = async (quantityInput: number | undefined) => {
 
   isLoading.value = true;
 
-  const response = await changeItemQuantity(Number(quantityInput));
-  // Refresh cart after qty update
-  await refreshCart(response);
+  try {
+    const response = await changeItemQuantity(Number(quantityInput));
+    // Refresh cart after qty update
+    await refreshCart(response);
+  } catch (error) {
+    const e = error as ClientApiError;
+    const { errors } = useApiErrorsResolver(e.messages);
```

Comment:
composables shouldn't be invoked in the runtime functions, proposed API:

```
const {resolveApiErrors} = useApiErrorsResolver();
```

and then in the function's block
```
 } catch (error) {
    const errors = resolveApiErrors(e.messages);
}
```